IntegrationTests/Runner/Environment.swift

```diff
@@ -45,16 +45,22 @@ internal struct URLSessionSetup: Codable {
     enum InstrumentationMethod: CaseIterable, Codable {
         /// Use `DDURLSessionDelegate` class directly.
         /// and define `firstPartyHosts` in feature configuration.
-        case directWithGlobalFirstPartyHosts
+        case directWithFeatureFirstPartyHosts
```

Comment:
question/ I can't see it at glance 👀 , what is the difference?

Reply:
Just renaming `Global` to `Feature`, there is no difference in behavior